sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -544,11 +593,23 @@ public ServiceBusReceiverClientBuilder queueName(String queueName) {
          *
          * @return The modified {@link ServiceBusReceiverClientBuilder} object.
          */
-        public ServiceBusReceiverClientBuilder receiveMode(ReceiveMode receiveMode) {
+        public ServiceBusSessionReceiverClientBuilder receiveMode(ReceiveMode receiveMode) {
             this.receiveMode = receiveMode;
             return this;
         }
 
+        /**
+         * Sets the session id.
+         *
```

Comment:
documentation: Setting this will get messages from this `sessionId` only and will **not** roll-over to next available session.